### ISSUE TITLE
Downloading error

### ISSUE THREAD

**Issue by TheBookishDad** [rmcrackan/Libation#1043]
**Describe the bug**

Click on a title to download and i get an error about abort retry or ignore the download. I was able to download after after we figured out my import issue. just not now.

**To Reproduce**

I tried multiple titles, changed download locations, multiple releases of libation, even deleted account file and started over

**Platform**

win 11

**Log Files**
[_Log202411.zip](https://github.com/user-attachments/files/17783503/_Log202411.zip)

**Comment by EdMcMillan** [comment]
I'm having the same issue. Attaching logs

[_Log202412_001.zip](https://github.com/user-attachments/files/18085194/_Log202412_001.zip)

[_Log202412.zip](https://github.com/user-attachments/files/18085189/_Log202412.zip)

**Comment by rmcrackan** [comment]
@EdMcMillan Can you download a fresh copy of the latest pre-release and see if you're still getting this -- https://github.com/rmcrackan/Libation/releases/tag/v11.6.3

**Comment by EdMcMillan** [comment]
I'll give it a shot and let you know - thank you!
    On Tuesday, December 10, 2024 at 03:20:45 PM EST, rmcrackan ***@***.***> wrote:  
 
 


@EdMcMillan Can you download a fresh copy of the latest pre-release and see if you're still getting this -- https://github.com/rmcrackan/Libation/releases/tag/v11.6.3

**Comment by EdMcMillan** [comment]
Hello!
I upgraded to 11.6.4 and completely started over.
Unfortunately, now Libation will not even import any of my Audible titles into the Libation library (vs before where the titles were listed in my library but I was unable to liberate them)
Please see attached log.  Many thanks
    On Wednesday, December 11, 2024 at 10:16:51 AM EST, ***@***.*** ***@***.***> wrote:  
 
  I'll give it a shot and let you know - thank you!
    On Tuesday, December 10, 2024 at 03:20:45 PM EST, rmcrackan ***@***.***> wrote:  
 
 


@EdMcMillan Can you download a fresh copy of the latest pre-release and see if you're still getting this -- https://github.com/rmcrackan/Libation/releases/tag/v11.6.3

**Comment by rmcrackan** [comment]
The log does not appear to have attached. Please try again.

**Comment by EdMcMillan** [comment]
Odd - I see it on my end, but in any event, I've extracted it from the zip file and attached the file directly.
Please let me know if it's still missing.
Many thanks,George
    On Saturday, February 15, 2025 at 03:44:03 PM EST, rmcrackan ***@***.***> wrote:  
 
  
The log does not appear to have attached. Please try again.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***>
rmcrackan left a comment (rmcrackan/Libation#1043)
The log does not appear to have attached. Please try again.

**Comment by wtanksleyjr** [comment]
Ed, try going to the GitHub web page and attaching the log from there. Your email system is completely jumbling everything about all your posts.

**Comment by EdMcMillan** [comment]
Thanks, @wtanksleyjr.

[_Log202502.zip](https://github.com/user-attachments/files/18813004/_Log202502.zip)

**Comment by rmcrackan** [comment]
@wtanksleyjr Thanks as always for the assist

@EdMcMillan Thank you for the log file. Since you're running windows, you can also try Windows Classic while I'm trying to debug this Windows Chardonnay bug.

@Mbucari I feel like we saw this in an older version of mac Avalonia but you already fixed it. Ideas?

```
System.PlatformNotSupportedException: Could not load ffmpegaac or ffmpegaac.arm64.dll
   at AAXClean.Codecs.Mp4FileExtensions.DllImportResolver(String libraryName, Assembly assembly, Nullable`1 searchPath) in /home/runner/work/AAXClean.Codecs/AAXClean.Codecs/src/AAXClean.Codecs/Mp4FileExtensions.cs:line 36
```

**Comment by Mbucari** [comment]
@rmcrackan 

> I feel like we saw this in an older version of mac Avalonia but you already fixed it. Ideas?

So I did fix this issue for mac, but it was a cheap workaround that I implemented in the workflow scripts for . The root cause in AaxClean.Codecs still hasn't been fixed, and I forgot that these days windows ALSO ships on ARM and has x64->arm translator. 

It's time I fix this for real. I've updated AAXClean.Codecs to 1.1.4, and this error SHOULD be fixed (although I have no way of testing since I don't own an arm cpu). I also updated the targeted .NET to include .net9.

**Comment by rmcrackan** [comment]
@Mbucari Thank you, sir!

@EdMcMillan Please let us know if this pre-release fixes your issue: https://github.com/rmcrackan/Libation/releases/tag/v11.6.5

**Comment by EdMcMillan** [comment]
@rmcrackan Thank you for your continued support. I updated to 12.0.1 on my PC and unfortunately Libation is still unable to load anything from my Audible library (please find the attached log) - thoughts?

[_Log202503.zip](https://github.com/user-attachments/files/19041102/_Log202503.zip)

**Comment by rmcrackan** [comment]
> Libation is still unable to load anything from my Audible library

Do you mean that nothing is showing up in Libation's grid?

**Comment by EdMcMillan** [comment]
Correct.  I entered my Audible credentials, as I did in November, but whereas before the books appeared in Libation (but were unable to download due to error messages) - my books no longer appear at all (which is the problem I've encountered since January) "No Books. Begin by Importing Your Library." in the lower right of the Libation homescreen after scanning.

**Comment by EdMcMillan** [comment]
It worked!  I just sent you a "literary tip" for all your help and your wonderful software.
Thank you and cheers.
    On Wednesday, March 5, 2025 at 08:55:38 AM EST, rmcrackan ***@***.***> wrote:  
 
  
This original bug seems to be out of date. If you have more information, please reply here and we can look into it further.

Please try the latest version: https://github.com/rmcrackan/Libation/releases/latest

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you were mentioned.Message ID: ***@***.***>
rmcrackan left a comment (rmcrackan/Libation#1043)
This original bug seems to be out of date. If you have more information, please reply here and we can look into it further.

Please try the latest version: https://github.com/rmcrackan/Libation/releases/latest
